package.json

```diff
@@ -60,7 +60,7 @@
     "@types/enzyme": "^3.10.3",
     "@types/jest": "^24.0.17",
     "@types/prop-types": "^15.7.1",
-    "@types/react": "^16.0.0",
+    "@types/react": "^16.9.2",

```

Comment:
@longlho we want to keep this at ^16.0.0 due to some bug right?

Reply:
`devDep` is ok I think